csrc/pf_text.c

```diff
@@ -35,11 +35,11 @@
 void pfReportError( const char *FunctionName, Err ErrCode )
 {
 	const char *s;
-	
+

```

Comment:
These white space errors have all been fixed.
So the code would need to be rebased and all tabs converted to 4 spaces.